src/renderers/0-3.js

```diff
@@ -140,7 +140,7 @@ export default class Renderer {
           const definedMarkup = this.markups.find(markup => markup.name === TagName);
           if (definedMarkup) {
             const { render: Markup } = definedMarkup;
-            pushElement(<Markup key={uuidV4()} {...props} />);
+            pushElement(<Markup key={uuidV4()} {...props} {...this.additionalProps} />);
```

Comment:
looks fine to me although you may want someone more familiar with mobiledoc to approve this before merging